ONSdigital/sdc-responses-dashboard
# Only load surveys with valid collection exercises

### Motivation and Context
We were passing all surveys to the view even when they did not have any collection exercises.

Since the user can only choose surveys that have at least 1 collection exercise, there is no need to send all surveys to the view.

### What has changed
Added an `if` to ensure the survey id must be in list of collection exercises.

### How to test?
- App works as expected.
- No change to functionality.
- Only surveys with collection is loaded in the view.

### Links




### Checklist

* [x] STATIC_ASSETS_VERSION updated.
